frontend/src/components/filters/descriptions.js

```diff
@@ -572,3 +572,46 @@ export const DATASET_FILE_FILTERS = {
     label: "Release Time",
   },
 }
+
+
+export const POOLED_SAMPLES_FILTERS = {
+    sample__id: {
+        // Note: The sample__id filter adds the pool id to the query.
+        // It's not mapped to any column, but is required for the pool id to be sent
+        // to the endpoint properly.
+      type: FILTER_TYPE.INPUT_OBJECT_ID,
+      key: "sample__id",
+    },
+    project_name: {
+        type: FILTER_TYPE.INPUT,
+        key: "derived_sample__project__name",
+        label: "Project Name"
+    },
+    alias: {
+      type: FILTER_TYPE.INPUT,
+      key: "derived_sample__biosample__alias",
+      label: "Alias"
+    },
+    volume_ratio: {
+      type: FILTER_TYPE.RANGE,
+      key: "volume_ratio",
+      label: "Volume Ratio"
+    },
+    library_type: {
+      type: FILTER_TYPE.INPUT,
+      key: "derived_sample__library__library_type__name",
+      label: "Library Type"
+    },
+    library_size: {
+      type: FILTER_TYPE.RANGE,
+      key: "derived_sample__library__library_size",
+      label: "Library Size",
+    },
+    index: {
```

Comment:
There is some information that are missing from the table that could be useful. Individual name (with link to individual), experimental group, collection site would be nice and sample kind (if there is some space left).

Reply:
Done